src/App.php

```diff
@@ -38,7 +38,7 @@ class App
 
     /** @var array|false Location where to load JS/CSS files */
     public $cdn = [
-        'atk' => 'https://ui.agiletoolkit.org/public', // develop branch
+        'atk' => 'https://cdn.jsdelivr.net/gh/atk4/ui@develop/public',
```

Comment:
keeping CDN instead of something like `/public`, as it will not work if not inside root or when using rewrite (in the future, we may use Symfony path parser for it)